content/variables/using-environment-variables.md

```diff
@@ -7,6 +7,28 @@ aliases:
 
 To access a variable, add the `$` symbol in front of its name. For example, access `API_TOKEN` by using `$API_TOKEN`. Note that it is required to use quotation marks with multi-line variables when you are referencing them in custom scripts.
 
+## Setting variables during the build
+
+By default, if you define an environment variable inside your script, you can only use it within the script itself. However, you can make an environment variable available to any subsequent parts of your workflow by defining or updating the environment variable and writing it to the `CM_ENV` environment file.
+
+Specifically, you can do this by writing a `"KEY=value"` pair to the `CM_ENV` environment file. `CM_ENV` can contain multiple environment variables separated by newlines. Instructions on how to write variables to the file can be found below.
+
+### Writing to an environment file on UNIX (macOS and Linux)
```

Comment:
Setting an environment variable on macOS or Linux

(It's not correct to use Unix)